tools/mason/MasonNew.chpl

```diff
@@ -91,13 +103,112 @@ proc masonNew(args) throws {
   }
 }
 
+/*
+  Starts an interactive session to create a
+  new library project.
+*/
+proc beginInteractiveSession(isInit: bool, defaultPackageName: string) throws {
+  writeln("This is an interactive session to walk you through creating a library ",
```

Comment:
Note that `"""multi-line strings"""` are available in Chapel, which may make this a little cleaner

Reply:
@ben-albrecht I did try to use that, but I had to remove whitespaces before the second and next lines which would break our indentation. Is there a way to ignore whitespaces  ?

Reply:
I'm working on it...  https://github.com/chapel-lang/chapel/pull/15664

See https://github.com/chapel-lang/chapel/issues/8281 for background if curious.

I would be OK breaking indentation for now, and we can come back and re-indent whenever `dedent` is available.